src/main/java/net/tylers1066/movecraftcannons/listener/RotationListener.java

```diff
@@ -21,12 +24,19 @@ public void rotateListener(CraftRotateEvent e) {
             return;
 
         Craft craft = e.getCraft();
-        if (!(craft instanceof PlayerCraft))
-            return;
+        UUID playerUUID = null;
+        if (craft instanceof PilotedCraft) {
+            playerUUID = ((PilotedCraft) craft).getPilot().getUniqueId();
+        } else if (craft instanceof SubCraft) {
+            if (((SubCraft) craft).getParent() instanceof PlayerCraft) {
+                playerUUID = ((PlayerCraft) ((SubCraft) craft).getParent()).getPilot().getUniqueId();
+            }
+        }
+
+        if (playerUUID == null) return;
```

Comment:
I think what we'll want to create here is a utility function for both rotation and translation that attempts to turn a `Craft` into a `UUID`:
if craft is null, return null
if craft is pilotedcraft, return pilot
if craft is subcraft, return recursive call to self with parent craft
else return null